script/build

```diff
@@ -10,16 +10,21 @@ fi
 which mono > /dev/null || (echo "mono must be installed"; exit 1)
 which mcs > /dev/null  || (echo "mcs must be installed"; exit 1)
 
-script/nuget install vendor/packages.config -o vendor
+bin/nuget install packages.config -o vendor
```

Comment:
Yeah, that's another oversight.  I really wanted executables in bin, but vendor turns out to be much more reasonable.  I'll fix it right up.